### ISSUE TITLE
agg = torch.cat([avg_attn, max_attn], dim=1) # spa b,2,h,w。  之后agg不应该经过conv+sigmoid吗？

### ISSUE THREAD

**Issue by dyysky** [YichuXu/DSFormer#8]
如题

**Comment by YichuXu** [comment]
Hi @dyysky,

Thanks a lot for interest to DSFormer.

We have re-uploaded our model DSFormer, and we sincerely appreciate your attention and support.

Best,
